docs/Design/Classes/Class Declarations.md

```diff
@@ -6,18 +6,22 @@ This improvement adds support for class declarations to GLS, including:
 
 * Naming a class
 * Declaring any generics *(optional)*
-* Declaring a class to extend *(optional)*
+* Declaring a parent class to extend *(optional)*
+* Declaring parent interfaces to implement *(optional)*
 
 
 ## Commands
 
 ### `class start`
 
-`class start` `:` `classDescriptor`*`[, parentClassDescriptor]`*
+`class start : classDescriptor [ "extends" parentClassDescriptor] [ "implements" parentInterfaceName1 parentInterfaceName2 parentInterfaceName3 ...] `
```

Comment:
Is there a reason you're using quotes here? I'm 70% sure I would prefer to not have them there. Some other place might have them with quotes, which should be fixed if so.